lumicks/pylake/kymotracker/detail/msd_estimation.py

```diff
@@ -447,3 +452,257 @@ def determine_optimal_points(frame_idx, coordinate, max_iterations=100):
         RuntimeWarning("Warning, maximum number of iterations exceeded. Returning best solution.")
     )
     return num_slope, num_intercept
+
+
+def _var_cve_unknown_var(
+    diffusion_constant: float,
+    variance_loc: float,
+    dt: float,
+    num_points: int,
+    blur_constant: float = 0,
+    avg_frame_steps: float = 1,
+) -> float:
+    """Expected variance of the diffusion estimate obtained with CVE.
+
+    The covariance-based diffusion estimator provides a simple unbiased estimator of diffusion.
+    This function is based on eqn 22 from Vestergaard et al [2] adapted for 1D. See also eqn 17 from
+    Vestergaard et al [1].
```

Comment:
Does this still align with the error message in `DiffusionEstimate`?  Do we still expect some bias when using cve as well? I'm referring to this line:
```HTML
        Note: while this provides some measure of uncertainty, these estimates are expected to be
        biased and should not be used to weight multiple diffusion estimates!
```

Reply:
Yes. The std error reported is biased. It's the best estimate we got at this point, but it's biased when inserting the estimates we obtained for `D` and `sigma` (I have also confirmed that there is a bias when simulating this). The only time we see no bias is when I compute it with the true values for `D` and `sigma`, but these will not be known when doing a real experiment. The paper also explicitly mentions that they should not be used to weight data. Look for `For importance weighting of estimates...` on page 7.

Reply:
I see. Then perhaps the wording can be altered to indicate the reasoning behind the note? The idea being providing enough information from the API without needing to bounce back and forth with the source. What do you think of the something like following?

`While this provides some measure of uncertainty, the estimate should be avoided when weighting multiple diffusion estimates.  This is to prevent complications such as bias due to correlations between the estimated parameters and the estimated variances`

Reply:
And maybe add the  suggestion from the author in the same paragraph 'when cal-
culating the weighted mean of estimates from time series
of different lengths—the length N of a time series should
be used as weight when possible, since it is known exactly'

Reply:
In the next PR, we'll be adding some functionality for actually weighting estimates coming from different tracks, so that the user never has to worry about this.

But yeah, I agree with both of these. How about for now we go with
```python
        Note: While this provides some measure of uncertainty, the estimate should not be used for
        weighting multiple diffusion estimates. This is to prevent complications such as bias due
        to correlations between the estimated parameters and estimated variances. Instead, when
        calculating the weighted mean of estimates from time series of different lengths—the length
        N of a time series should be used as weight when possible, since it is known exactly.
```

And then once we have the functionality on the group, we refer to the group functions there (e.g. note that you can compute diffusion constants for a group of tracks in the following way etc etc....).